### DISCUSSION TITLE
classes/class_animationrootnode

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#173] [User-contributed notes]
# classes/class_animationrootnode

Inherits: AnimationNode< Resource< RefCounted< Object Inherited By: AnimationNodeAnimation, AnimationNodeBlendSpace1D, AnimationNodeBlendSpace2D, AnimationNodeBlendTree, AnimationNodeStateMachine B...

https://docs.godotengine.org/en/stable/classes/class_animationrootnode.html

**Comment by GitHub7796** [comment]
I guess AnimationRootNode should be an abstract class
